lib/Bindings/Python/circt/dialects/_hw_ops_ext.py

```diff
@@ -87,7 +88,7 @@ def set_input_port(self, name, value):
       # Put the value into the instance.
       index = self.operand_indices[name]
       self.instance.inputs[index] = value
-      self.parent_module.remove_backedge(self.backedges[index])
+      self.backedges[index].erase()
```

Comment:
note to self: this logic is being moved to `connect`, and I'll want to rebase #1106 on this.